rinex/src/antex/antenna.rs

```diff
@@ -21,14 +22,8 @@ pub enum CalibrationMethod {
     Converted,
 }
 
-impl Default for CalibrationMethod {
-    fn default() -> Self {
-        Self::Chamber
-    }
-}
-
 /// Calibration information
-#[derive(Clone, Debug, PartialEq, PartialOrd)]
+#[derive(Default, Clone, Debug, PartialEq, PartialOrd)]
```

Comment:
The default `agency` and `date` are now empty strings instead of `"Unknown"`, right? Just making sure it's intended.

Reply:
@lnicola yes, I thought about is and I'm not sure what the best solution is.

At the time I created this, I thought that having "Unknown" (ie., not an empty string) would be more useful to the end user, or to determine what data fields could be filled.

This only serves data production (file creation) and will not impact the parsing mode